lib/Installation/EncryptedVolume/EncryptedVolumeController.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Controller for YaST Encrypted Volume module.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::EncryptedVolume::EncryptedVolumeController;
+use strict;
+use warnings;
+use YuiRestClient;
+use Installation::EncryptedVolume::EncryptedVolumePage;
```

Comment:
IMO it should be `Installation::SystemProbing::EncryptedVolumeActivationPage` (with SystemProbing as folder) as this screen could contains other popups and it is clear by the name that belongs to that phase of system probing by YaST.

Reply:
Done